src/drivers/stomp/Queue.php

```diff
@@ -93,6 +93,12 @@ class Queue extends CliQueue
      */
     public $commandClass = Command::class;
 
+    /**
+     * Set the read timeout.
+     * @var float
+     */
+    public $readTimeOut = 0.1;
```

Comment:
The timeout is in milliseconds, so 0.1 means 100 microseconds. Is it really something you want it to be?

Reply:
One second is 1000

Reply:
readTimeOut in seconds
look at 
https://github.com/php-enqueue/stomp/blob/master/BufferedStompClient.php#L55
https://github.com/php-enqueue/stomp/blob/master/StompConsumer.php#L101

Reply:
by the way you are right, receive function need a INT argument

Reply:
yeah, in 0.8 stomp uses a timestamp in seconds, which is not right according to interface and is not align with other transports.